_site/public/tutorials/pet-shop.md

```diff
@@ -4,484 +4,594 @@
 
 # Ethereum Pet Shop
 
-This series of tutorials will take you through building your first DApp--- a tracking system for a pet shop!
+This series of tutorials will take you through building your first dapp---an adoption tracking system for a pet shop!
```

Comment:
So we're not calling them `ðapps`? 😜 
([Ð wiki](https://en.wikipedia.org/wiki/Eth))

Reply:
If we did that, we'd have to start calling it Eþereum. ;)